### PR TITLE
Add the boot option inst.xtimeout

### PR DESCRIPTION
**Pull request by poncovka**
Added a new boot option inst.xtimeout to set the timeout in seconds
for starting X.

(cherry-picked from a commit 08afa5c)

### CODE REVIEW

**Comment by poncovka** [comment]
Resolves the issue https://github.com/rhinstaller/anaconda/issues/1237

File: data/anaconda_options.txt [hunk 1]
```diff
@@ -107,6 +107,9 @@ the password needs to be 6 to 8 characters long (limitation of the VNC protocol)
 xdriver
 Use DRIVER as the X driver to use during installation as well as on the installed system.
 
+xtimeout
+Specify the timeout in seconds for starting X.
```
Comment (jkonecny12) [inline data/anaconda_options.txt]: I would add `X server` to be more precise.

File: docs/boot-options.rst [hunk 2]
```diff
@@ -480,6 +480,13 @@ Use the framebuffer X driver (``fbdev``) rather than a hardware-specific driver.
 
 Equivalent to ``inst.xdriver=fbdev``.
 
+.. inst.xtimeout:
+
+inst.xtimeout
+^^^^^^^^^^^^^
+
+Specify the timeout in seconds for starting X.
```
Comment (jkonecny12) [inline docs/boot-options.rst]: Same here.

File: data/anaconda_options.txt (hunk 1 above)
Comment (poncovka) [reply data/anaconda_options.txt]: Fixed.

File: docs/boot-options.rst (hunk 2 above)
Comment (poncovka) [reply docs/boot-options.rst]: Fixed.

File: pyanaconda/iutil.py [hunk 3]
```diff
@@ -48,7 +48,7 @@
 from gi.repository import GLib
 
 from pyanaconda.flags import flags
-from pyanaconda.constants import DRACUT_SHUTDOWN_EJECT, TRANSLATIONS_UPDATE_DIR, UNSUPPORTED_HW, IPMI_ABORTED
+from pyanaconda.constants import DRACUT_SHUTDOWN_EJECT, TRANSLATIONS_UPDATE_DIR, UNSUPPORTED_HW, IPMI_ABORTED, X_TIMEOUT
```
Comment (jkonecny12) [inline pyanaconda/iutil.py]: One last small thing, could you please wrap this line to 99 characters before merge. Thanks.

**Comment by jkonecny12** [comment]
It looks even better now :).

File: pyanaconda/iutil.py (hunk 3 above)
Comment (poncovka) [reply pyanaconda/iutil.py]: Fixed.

**Pull request merged**